tests/__main__.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
Yay unittests!
Die landen aber nicht im Image oder? Nicht, dass wir da Platzprobleme bekommen.

Reply:
Da die nicht im `deploy.sh` in `FILELIST` drin stehen, sollten die nicht mit eingepackt werden.